docs/deployment/cognito/README.md

```diff
@@ -32,16 +32,14 @@ This guide assumes that you have:
         --nodes-max 10 \
         --managed
         ```
-1. AWS IAM permissions to create roles and attach policies to roles.
 
-1. Clone the `awslabs/kubeflow-manifest` repo, `kubeflow/manifests` repo and checkout the desired branches
-    1. Substitute the value for `KUBEFLOW_RELEASE_VERSION`(e.g. v1.4.1) and `AWS_MANIFESTS_BUILD`(e.g. v1.4.1-b1.0.0) with the branch or tag you want to use below
+1. Clone the `awslabs/kubeflow-manifest` repo, `kubeflow/manifests` repo and checkout the release branches.
+    1. Substitute the value for `KUBEFLOW_RELEASE_VERSION`(e.g. v1.4.1) and `AWS_RELEASE_VERSION`(e.g. v1.4.1-aws-b1.0.0) with the tag or branch you want to use below. Read more about [releases and versioning](../../community/releases.md#releases-and-versioning) policy if you are unsure about what these values should be.
```

Comment:
[nit] can we change [line 37] numbered list to bullet list just like in [line 19 and 20] to be consistent